### PR TITLE
Enable test.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Review by hrkrshnn — DISMISSED** [review]

**Review by leonardoalt — DISMISSED** [review]

**Comment by chriseth** [comment]
That was a bit too quick.

File: test/libsolidity/SolidityEndToEndTest.cpp [hunk 1]
```diff
@@ -1381,6 +1381,8 @@ BOOST_AUTO_TEST_CASE(internal_constructor)
 			constructor() {}
 		}
 	)";
+	// Disabled via yul because it will throw an error instead of
+	// returning empty bytecode.
```
Comment (leonardoalt) [inline test/libsolidity/SolidityEndToEndTest.cpp]: Where was it disabled?

Comment (chriseth) [reply test/libsolidity/SolidityEndToEndTest.cpp]: It was not enabled. All other tests here can be enabled.

**Comment by chriseth** [comment]
rebased.

File: test/libsolidity/SolidityEndToEndTest.cpp (hunk 1 above)
Comment (leonardoalt) [reply test/libsolidity/SolidityEndToEndTest.cpp]: ah ok

Comment (leonardoalt) [reply test/libsolidity/SolidityEndToEndTest.cpp]: This comment vanished now, was that intended?

Comment (chriseth) [reply test/libsolidity/SolidityEndToEndTest.cpp]: It was part of another PR that got merged in the meantime.

**Pull request merged**